assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js

```diff
@@ -0,0 +1,25 @@
+var WAIT_TIMEOUT = 5000;
+
+module.exports = function(casper, scenario) {
+  var waitFor = require('./waitForHelperHelper')(casper, WAIT_TIMEOUT);
```

Comment:
I think you can replace your var by let/const

Reply:
Oh all of these are not mine 😸

Reply:
Maybe you could put this in a `vendor/` directory. I think that's the convention for external libraries.